SpinCore_pp/ppg/generic.py

```diff
@@ -32,6 +31,7 @@ def generic(
     ph2_cyc=r_[0],
     ret_data=None,
     amplitude=1.0,
```

Comment:
do we want `amplitude=1.0` if we may soon use this with the new probe? Why not just include amplitude as an argument without presetting it to 1?

Reply:
amplitude is also not included in the description of each parameter below

Reply:
when you define a keyword argument, you have to give it a default.